cli/integration-test/integration/fixture.clj

```diff
@@ -12,27 +12,47 @@
      :params params
      :id (lsp/inc-request-id)}))
 
+(def default-init-options {:lint-project-files-after-startup? false})
+(def emacs-capabilities
```

Comment:
I don't think we need to have capabilities specific for a client like Emacs, AFACS we just use it for code actions bring a lot of actions similar to emacs, but I think we could create different tests acvating different capabilities just like the code actions unit tests do, WDYT?

Reply:
I added this because I realized the code action integration tests were getting different responses than Emacs was. It took me a while to figure out that's because the tests default to the minimum client capabilities. Once I understood that it took awhile to track down all the possible capabilities, and then another while to figure out what Emacs was actually using, and then another while to figure out how to write the JSON to turn them on.

So, I sort of like being able to look in one place to see how to turn on all the client capabilities.

But, you're right, this is only used in the code action tests. Maybe it'd be better to inline it into that test? Or maybe change it from `emacs-capabilities` to `full-client-capabilities`, so that it's easy to write other tests for fully featured clients? What do you prefer?

Reply:
Or to put it another way, when I'm writing integration tests, I want to pretend I don't know anything about the code. I want to say "pretend we're in an editor..." not, "pretend we're in an editor which has a very specific set of capabilities that I know will affect the outcome of this test..." That's a job for unit tests, IMO.

Reply:
I see your point, I think most of the actions just rely on `[:workspace :workspace-edit]` which is a pretty common capability almost (all?) editors have nowadays, I'd still go with just enabling it for the actions we would like to get testing both ways, but having it only on a code actions test is a way too.

Reply:
Makes sense, I'll update it.